Formula/grass.rb

```diff
@@ -38,20 +37,14 @@ class Grass < Formula
   depends_on "unixodbc"
   depends_on "fftw"
   depends_on "cairo"
-  depends_on "freetype"
   depends_on :x11 # needs to find at least X11/include/GL/gl.h
-  depends_on "wxpython" => :recommended
+  depends_on "wxpython" if build.with? "gui"
   depends_on :postgresql => :optional
   depends_on :mysql => :optional
 
-  fails_with :clang do
-    cause "Multiple build failures while compiling GRASS tools."
-  end
-
   def headless?
-    # The GRASS GUI is based on WxPython. Unfortunately, Lion does not include
-    # this module so we have to drop it.
-    build.without?("gui") || MacOS.version == :lion
+    # The GRASS GUI is based on WxPython.
+    build.without? "gui"
```

Comment:
The check is now sufficiently simple and no longer warrants a helper method. Please inline this code.